### PR TITLE
chore: adjust copyright header

### PR DESCRIPTION
**Pull request by FrankEssenberger**
adjust copyright header

### CODE REVIEW

File: .github/actions/get-changelog/dist/.github/actions/get-changelog/index.js [hunk 1]
```diff
@@ -1,10 +0,0 @@
-"use strict";
```
Comment (jjtang1985) [inline .github/actions/get-changelog/dist/.github/actions/get-changelog/index.js:1]: I guess the changes under the actions should not be touched.

**Review by jjtang1985 — CHANGES_REQUESTED** [review]
I guess the changes under the actions should not be touched.

File: .github/actions/get-changelog/dist/.github/actions/get-changelog/index.js (hunk 1 above)
Comment (FrankEssenberger) [reply .github/actions/get-changelog/dist/.github/actions/get-changelog/index.js]: good that you spotted this. accidental delete - I tried to revert, but I got two formatting changes I can get rid off.

Comment (FrankEssenberger) [reply .github/actions/get-changelog/dist/.github/actions/get-changelog/index.js]: git says the files are the same.

**Review by jjtang1985 — APPROVED** [review]
Look good.

**Pull request merged**